### DISCUSSION TITLE
8월2일 피드백 내용

### DISCUSSION THREAD

**Discussion by jeongmino1** [CodeMedics/Harulog-backend#3] [General]
피드가 직관적이지 않음, 차별성 필요, 서로 같은 감정의 사람들 연결?
피드를 볼 때 제목이 보이게
최신순 인기순 팝업창 노필요해보임
피드에 닉네임 보이게
사용자 평가 페이지의 목적
비밀글은 관리자가 볼 수 있게 할 것인지
페이징 처리
모달보다는 상세페이지를 따로 만들어라?
초기 인원모집이 어려워 보임
기분별로 쓴 글 보이게
위로와 공감을 해줄 수 있는 기능 필요
달력 형식으로  UI를 보여주면 좋을 것 같다
관종 아님 정신이상자가 사용할 듯 -> 초기 컬쳐가 중요
상담기능?
한 달의 기분을 볼 수 있게 하면 좋을 것 같다
글에 대한 필터링 필요 -> 카테고리
이슈 관련 페이지 필요? -> 일기가 아닌 다른 얘기를 할 가능성이 높음
사용자의 기분을 꼭 색깔만이 아닌 다른 형식으로도 고려
매뉴 버튼 위치, 가독성
익명으로 하면 위험성이 높음 
우울한 사람들이 모이면 굉장히 위험 -> 온라인이라도
컨셉을 좀 더 밝게 했으면 좋겠음

**Comment by Oleevia01** [comment]
피드백 참고해서 컨셉 정리하는데 많은 도움이 되었습니다🙂
